rust/worker/src/execution/orchestration/hnsw.rs

```diff
@@ -619,6 +636,25 @@ impl Handler<TaskResult<PullLogsOutput, PullLogsError>> for HnswQueryOrchestrato
         match message {
             Ok(pull_logs_output) => {
                 let logs = pull_logs_output.logs();
+                // Divide the allowed_ids into two mutually exclusive lists
```

Comment:
not needed now but should this somehow be part of the chunk abstraction or a wrapper - as this seems trivial to push higher up

Reply:
agreed